cloud_service/getting-started-rhacs-cloud-ocp.adoc

```diff
@@ -0,0 +1,104 @@
+====
+
+[id="installation-overview-acs-cloud"]
+== High-level overview of installation steps
+
+The following sections provide an overview of installation steps and links to the relevant documentation.
+
+[id="securing-rh-cloud-clusters"]
+=== Securing {osp} clusters
+
+To secure {osp} clusters by using the Operator, perform the following steps:
+
+. Verify that the clusters you want to secure meet the xref:../cloud_service/installing_cloud_ocp/prerequisites-cloud-ocp.adoc#prerequisites-cloud-ocp[prerequisites].
+. In the {cloud-console}, xref:../cloud_service/installing_cloud_ocp/cloud-create-instance-ocp.adoc#cloud-create-instance-ocp[create an *ACS Instance*].
+. On each {osp} cluster you want to secure, xref:../cloud_service/installing_cloud_ocp/cloud-ocp-create-project.adoc#cloud-ocp-create-project[create a project named `stackrox`]. This project will contain the resources for {product-title-managed-short} secured clusters.
+. In the ACS Console, xref:../cloud_service/installing_cloud_ocp/init-bundle-cloud-ocp-generate.adoc#init-bundle-cloud-ocp-generate[create an init bundle]. The init bundle contains secrets that allow communication between {product-title-managed-short} secured clusters and the ACS Console.
+. On each {osp} cluster, xref:../cloud_service/installing_cloud_ocp/init-bundle-cloud-ocp-apply#init-bundle-cloud-ocp-apply[apply the init bundle] by using it to create resources.
+. On each {osp} cluster, xref:../cloud_service/installing_cloud_ocp/cloud-install-operator#cloud-install-operator[install the {product-title-short} Operator].
+. On each {osp} cluster, xref:../cloud_service/installing_cloud_ocp/install-secured-cluster-cloud-ocp.adoc#installing-sc-operator-cloud-ocp[install secured cluster resources in the `stackrox` project] by using the Operator.
+. xref:../cloud_service/installing_cloud_ocp/verify-installation-cloud-ocp.adoc#verify-installation-cloud-ocp[Verify installation] by ensuring that your secured clusters can communicate with the ACS instance.
+
+To secure {osp} clusters by using Helm charts or the `roxctl` CLI, perform the following steps:
+
+. . Verify that the clusters you want to secure meet the xref:../cloud_service/installing_cloud_ocp/prerequisites-cloud-ocp.adoc#prerequisites-cloud-ocp[prerequisites].
+. In the {cloud-console}, xref:../cloud_service/installing_cloud_ocp/cloud-create-instance-ocp.adoc#cloud-create-instance-ocp[create an *ACS Instance*].
+. On each {osp} cluster you want to secure, xref:../cloud_service/installing_cloud_ocp/cloud-ocp-create-project.adoc#cloud-ocp-create-project[create a project named `stackrox`]. This project will contain the resources for {product-title-managed-short} secured clusters.
```

Comment:
I know we generally like to use present tense, but I think since the project is created at this point, but doesn't contain any files yet, it's important to have the context (from "will") that it will store the files that are installed later.